e2e/testsuite/testsuite.go

```diff
@@ -105,6 +105,25 @@ func newPath(chainA, chainB *cosmos.CosmosChain) path {
 	}
 }
 
+// relayerMap is a mapping from test names to a relayer set for that test.
+type relayerMap map[string]map[ibc.Wallet]bool
+
+// addRelayer adds the given relayer to the relayer set for the given test name.
+func (r relayerMap) addRelayer(testName string, relayer ibc.Wallet) {
+	if _, ok := r[testName]; !ok {
+		r[testName] = make(map[ibc.Wallet]bool)
+	}
+	r[testName][relayer] = true
+}
+
+// containsRelayer returns true if the given relayer is in the relayer set for the given test name.
+func (r relayerMap) containsRelayer(testName string, wallet ibc.Wallet) bool {
+	if relayerSet, ok := r[testName]; ok {
+		return relayerSet[wallet]
+	}
+	return false
+}
```

Comment:
should this go in `relayer/relayer.go`? Seems like a convenience struct we can hide in the background

Reply:
Sound good to me, done in fb515f5